src/lib/InputComponent.tsx

```diff
@@ -294,13 +297,13 @@ export class InputComponent extends React.Component<Props, State> {
             testID={this.props.testID ?? "Input"}
             keyboardType={this.props.keyboardType}
             style={this.props.inputStyle}
-            onChange={this.handleChangeFromInput}
+            onChange={onChange}
             onContentSizeChange={this.handleContentSizeChange}
             onFocus={this._scrollToInput}
             placeholder={this.props.placeholder}
             value={this.state.displayValue}
```

Comment:
isn't there an `editable` prop which could be set to `!readOnly` instead?
it might be slightly better in that a keyboard would be prevented from appearing and there would be no flicker when typing a char

Reply:
I see you tried this in an earlier commit, although it wasn't set according to the readonly prop

Reply:
The issue with the editable prop is it stops the user from selecting and copying the value. Readonly should selectable. I couldn't see a way without making native changes (adding a package which adds a selectable `Text` text component) to make the text selectable, but not changeable

Reply:
I just tried with a different text input and I see that you can't select text when editable is false. So I'll approve this PR